spng-sys/build.rs

```diff
@@ -13,8 +13,5 @@ fn main() {
     }
     build.compile("spng");
 
```

Comment:
Please add this back with the updated path. It exposes the spng include directory to downstream cargo crates in the event that you need to depend on spng from another native/sys build.

```rust
    // DEP_SPNG_INCLUDE
    println!("include=libspng/spng");
```

Reply:
Done in the latest commits, thanks.